dashboard/db/schema.rb

```diff
@@ -426,6 +426,10 @@
     t.string "marketing_initiative"
     t.string "grade_levels"
     t.string "header"
+    t.string "image"
+    t.string "cs_topic"
+    t.string "school_subject"
+    t.string "device_compatibility"
```

Comment:
the best practice is for migrations to be their own PR, including the effects of running them (updates to model comments and schema.rb). the migration PR should then be merged ahead of the code changes which depend on them. this is to avoid potential problems if the PR needs to be rolled back. 

the way I normally go about this is to make the migration be the first commit(s) in my feature branch, and then once I have everything working, go back and create a new branch for the migration PR based off of the first commit, and send both PRs for review at the same time. happy to step through this next time you are facing a migration if needed. 

for this time, I would say not to bother refactoring this PR, and maybe just plan to merge it early enough in the day / week that we have time to respond to any issues during business hours.

Reply:
Oh I see, thank you that makes sense! I will keep that in mind for the future!